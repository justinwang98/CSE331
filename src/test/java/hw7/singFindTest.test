# Load the songs dataset and search for an existing path.

LoadGraph songs songs.tsv

FindPath songs JJLin AoiEir
