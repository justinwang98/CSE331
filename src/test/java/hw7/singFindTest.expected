# Load the songs dataset and search for an existing path.

loaded graph songs

path from JJLin to AoiEir:
JJLin to Lisa with weight 1.000
Lisa to AoiEir with weight 1.000
total cost: 2.000